### ISSUE TITLE
Add ontology documentation

### ISSUE THREAD

**Issue by agbeltran** [FAIR-IMPACT/MOD#41]
Consider adding ontology documentation (e.g. Widoco) in an automated workflow (i.e. GitHub actions)

**Comment by jonquet** [comment]
Yes. Once we have "converged" the mod-v2.0_ontology and mod-v2.0_profile into one unique formalisation for v2.1 then we can work on regenerating the documentation and automating this in GitHub. 
Do you know how to GitHub action this? shall we ask @dgarijo

**Comment by dgarijo** [comment]
I do it through OnToology http://ontoology.linkeddata.es/ (usually easier).

**Comment by jonquet** [comment]
> I do it through OnToology http://ontoology.linkeddata.es/ (usually easier).

Perfect. We shall contact in time to do it ;)

**Comment by jonquet** [comment]
@antony-wilson Could you explain us there if it works now and how? I see a widoco GitHub action on the MOD repo... but not sure if the pipeline is working so we get widoco generated doc each time we comit a new version...
